### PR TITLE
Update requirements.txt to fix failing

### PR DESCRIPTION
**Pull request by Tearran**
Also, update the installation command in the README.md file to use "--use-pep517" flag for pip install.

### CODE REVIEW

**Comment by EvilOlaf** [comment]
What are the changes besides requirements all about?

**Comment by Tearran** [comment]
@EvilOlaf  I Ran `python3 tools/mkArmbianDocs.py && mkdocs build`  to confirm it changed the mkdocs.yml"

My changes

In the readme
libsass had deprecated error during instal to fix `--use-pep517` ref. https://github.com/pypa/pip/issues/8559

in `tools/mkArmbianDocs.py`
deprecated warning for showed fix in output during build 

as for requirements.txt I just blindly updated versions To see if would build.
 
Today found fix for deprecated mkdocs-with-pdf warning by adding weasyprint==44 to requirement.txt

with the changed all deprecated messages are gone during `mkdoc build `

pastebin output , https://paste.armbian.com/xogubumume.bash

File: mkdocs.yml [hunk 1]
```diff
@@ -1,34 +1,35 @@
+
 site_name: Armbian Documentation
 site_author: "Armbian team"
 
-copyright: Copyright &copy; 2015 - 2024 Armbian team
+copyright: Copyright &copy; 2015 - 2022 Armbian team
```
Comment (EvilOlaf) [inline mkdocs.yml:5]: why?

File: mkdocs.yml [hunk 2]
```diff
@@ -41,17 +42,15 @@ extra:
...
 plugins:
-  - search
-  - with-pdf:
-      author: Armbian documentation team
-      copyright: © 2024 by Armbian
-      cover_title: Armbian documentation
-      cover_subtitle: Linux for ARM development boards
+   - with-pdf:
+       author: Armbian documentation team
+       copyright: © 2021 by Armbian
```
Comment (EvilOlaf) [inline mkdocs.yml:47]: why?

File: mkdocs.yml [hunk 3]
```diff
@@ -60,61 +59,74 @@ markdown_extensions:
...
   - pymdownx.superfences
   - pymdownx.emoji:
-      emoji_index: !!python/name:materialx.emoji.twemoji
-      emoji_generator: !!python/name:materialx.emoji.to_svg
+      emoji_index: !!python/name:material.extensions.emoji.twemoji
+      emoji_generator: !!python/name:material.extensions.emoji.to_svg
 
 nav:
   - Home: index.md
+  - 'Board' :
```
Comment (EvilOlaf) [inline mkdocs.yml:67]: why did the sorting change?

File: mkdocs.yml (hunk 1 above)
Comment (Tearran) [reply mkdocs.yml:5]: Why did this change? Perhaps someone manually edited the date instead or fixing the tool that generated this file.

File: mkdocs.yml (hunk 3 above)
Comment (Tearran) [reply mkdocs.yml:67]: To reiterate the file was not manually edited. It was generated 'python3 tools/mkArmbianDocs.py'

**Comment by Tearran** [comment]
made changes to `tools/mkArmbianDocs.py` to fix date and add missing `- search`

File: mkdocs.yml (hunk 3 above)
Comment (Tearran) [reply mkdocs.yml:69]: The code that does this is Lines 144 - 147 https://github.com/armbian/documentation/blob/d49213ebfb6de8273b145a00b79d5f93c0f4c917/tools/mkArmbianDocs.py#L144

**Comment by EvilOlaf** [comment]
Nice.

@igorpecovnik  objections?

**Comment by igorpecovnik** [comment]
None. Just a note that PDF generation fails to complete, but not sure if it has something to do with those changes.

**Comment by igorpecovnik** [comment]
Thank @Tearran PDF generation works again.

**Comment by igorpecovnik** [comment]
I need to add this
`pip install mdx-truly-sane-list`
otherwise it failed to build.

**Pull request merged**